diffusers_doc/en/write_own_pipeline.ipynb

```diff
@@ -53,7 +53,7 @@
     "\n",
```

Comment:
<p>"The pipeline begins with random noise that has the same shape as the desired output, and then passes it through the model according to the number of steps set to run the denoising process for."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>